### PR TITLE
Update advocacy.code.org homepage and add /coalition page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Some misc updates to https://advocacy.code.org:
- Moves the logos at the bottom of the current homepage to https://advocacy.code.org/coalition
  -  Also adds a link to the new Coalition page in the homepage hero banner
- Updates the call to action boxes on the homepage
  - Goes from 4 to 3, updates the links and styling a bit
- Updates some styles on the State of CS and TeachAI banners to make them look a little bit better
- Page is fully responsive, and not translated since this is a US-based audience.

_Notes:_ 
- The `advocacy-home.css` stylesheet got linted so there are some irrelevant diffs, but nothing major. Also, some of the css updates here are hacky, but could be improved if we decide to redesign/do a major refactor.
- I updated the "Back to Code.org" button in the top right nav in Dropbox to look how it does in the screenshots, but locally it will look huge and purple since it hasn't been scooped yet.

**Jira ticket:** [ACQ-1489](https://codedotorg.atlassian.net/browse/ACQ-1489)

---- 

## advocacy.code.org homepage
localhost-advocacy code org_3000_

## New advocacy.code.org/coalition page
localhost-advocacy code org_3000_coalition

### CODE REVIEW

File: pegasus/sites.v3/advocacy.code.org/public/css/advocacy-home.css [hunk 1]
```diff
@@ -15,7 +15,10 @@
   color: #666;
 }
 
-.info a:link, .info a:visited, .states-link a:link, .states-link a:visited {
```
Comment (hannahbergam) [inline pegasus/sites.v3/advocacy.code.org/public/css/advocacy-home.css:18]: It looks like we've got a good bunch of linter updates yeah?

File: pegasus/sites.v3/advocacy.code.org/public/index.haml [hunk 2]
```diff
@@ -18,9 +18,11 @@ video_player: false
 - banner_body = capture_haml do
   Bringing together 100 industry, nonprofit, and advocacy organizations, the Code.org Advocacy Coalition is growing the movement to make computer science a fundamental part of K-12 education.
 
-  .join-now
-    %a.link-button{href: "https://bit.ly/joinadvocacycoalition", target: "_blank"}
+  .button-wrapper
+    %a.link-button.has-external-link{href: "https://bit.ly/joinadvocacycoalition", target: "_blank", rel: "noopener noreferrer"}
```
Comment (hannahbergam) [inline pegasus/sites.v3/advocacy.code.org/public/index.haml:22]: For my own learning: what does .has-external-link do?

File: pegasus/src/advocacy_site.rb [hunk 3]
```diff
@@ -77,47 +57,11 @@ def self.get_blocks(request)
...
             },
-            {
-              text_actual: "More policy resources",
-              url: "/policy-resources"
-            },
-
           ]
       },
-
-      {
```
Comment (hannahbergam) [inline pegasus/src/advocacy_site.rb:97]: We love the negative lines

**Review by hannahbergam — APPROVED** [review]
LGTM, but I'm going to look more closely at the before/after!

File: pegasus/sites.v3/advocacy.code.org/public/css/advocacy-home.css (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/advocacy.code.org/public/css/advocacy-home.css:18]: I think there's just this one, and line 120(old)/173(new).

File: pegasus/sites.v3/advocacy.code.org/public/index.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/advocacy.code.org/public/index.haml:22]: This class adds the little box arrow icon on buttons and text links that open in a new tab 😁https://github.com/code-dot-org/code-dot-org/blob/f76661a133b2661fe8ef55463e7b4fcd308d2ec4/pegasus/sites.v3/code.org/styles_min/040-page.css#L344-L358

**Pull request merged**